docs/en/end-user/components/more.md

```diff
@@ -4,6 +4,48 @@ title:  Want More?
 
 Components in KubeVela are designed to be brought by users.
 
+## 1. Get from capability center
+
+KubeVela allows you to explore capabilities maintained by other developer.
+There are two command in kubectl vela plugin: `comp` and `trait`.
+
+In case you haven't install kubectl vela plugin: see [this](../../kubectl-plugin).
+
+### 1. list
+
+For example, let's try to list all availible components in registry:
+
+```shell
+$ kubectl vela comp --discover
+Showing components from default registry:https://github.com/oam-dev/catalog/tree/master/registry
+NAME              	CENTER	DEFINITION                 	
+cloneset          	default	clonesets.apps.kruise.io
+kruise-statefulset	default	statefulsets.apps.kruise.io
+openfaas          	default	functions.openfaas.com
+````
+Note that the `--discover` flag means show all uninstalled components.
+
+### 2. install
+Then you can install component like:
+
+```shell
+$ kubectl vela comp get cloneset
```

Comment:
Why don't we use `install` instead of `get`?

Reply:
yeah, that's also @wonderflow's idea, I think they were both OK. `get`  seems smaller.

Reply:
using `get` can help us to tell that this is very small and lightweight operation

Reply:
using `get` can help us to tell that this is very small and lightweight operation